openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -25,6 +25,7 @@
     'logo': "https://edly-edx-theme-files.s3.amazonaws.com/st-lutherx-logo.png",
     'favicon': "https://edly-edx-theme-files.s3.amazonaws.com/favicon.ico",
 }
+DEFAULT_EDLY_COPYRIGHT_TEXT = 'Edly 2020. All rights reserved.'
```

Comment:
This text should be `© Edly {current_year}. All rights reserved.`

Reply:
Feedback Implemented